<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureViewer</class>
 <widget class="QDialog" name="TemperatureViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clean up previsualization</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="cleanDataTab">
      <property name="maximumSize">
       <size>
        <width>797</width>
        <height>559</height>
       </size>
      </property>
      <attribute name="title">
       <string>Raw Data Cleanup</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>22</y>
         <width>781</width>
         <height>531</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="6" column="3">
         <widget class="QRadioButton" name="radioButtonNone">
          <property name="text">
           <string>None</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupMethod</string>
          </attribute>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QCheckBox" name="checkBoxFilter">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QCheckBox" name="checkBoxChanges">
          <property name="text">
           <string>Show changes</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QVBoxLayout" name="widgetToolBar"/>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="labelMethod">
          <property name="text">
           <string>Outliers method:</string>
          </property>
         </widget>
        </item>
        <item row="16" column="3">
         <widget class="QPushButton" name="pushButtonResetVar">
          <property name="text">
           <string>Reset variable</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditShowPoint">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QComboBox" name="comboBoxRawVar"/>
        </item>
        <item row="17" column="3">
         <widget class="QPushButton" name="pushButtonResetAll">
          <property name="text">
           <string>Reset all</string>
          </property>
         </widget>
        </item>
        <item row="11" column="3">
         <widget class="QPushButton" name="pushButtonSelectPoints">
          <property name="text">
           <string>Select points</string>
          </property>
         </widget>
        </item>
        <item row="12" column="3">
         <widget class="QPushButton" name="pushButtonEditCode">
          <property name="text">
           <string>Edit code</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QRadioButton" name="radioButtonZScore">
          <property name="text">
           <string>Z-score</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupMethod</string>
          </attribute>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QRadioButton" name="radioButtonIQR">
          <property name="text">
           <string>IQR</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroupMethod</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonLoadRawData">
          <property name="text">
           <string>Load Raw Data</string>
          </property>
         </widget>
        </item>
        <item row="10" column="3">
         <widget class="QLabel" name="labelAdvanced">
          <property name="text">
           <string>Advanced:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" rowspan="14" colspan="3">
         <layout class="QVBoxLayout" name="widgetRawData"/>
        </item>
        <item row="15" column="3">
         <widget class="QCheckBox" name="checkBoxCK">
          <property name="text">
           <string>From ºC to K</string>
          </property>
         </widget>
        </item>
        <item row="14" column="3">
         <widget class="QCheckBox" name="checkBoxFC">
          <property name="text">
           <string>From ºF to ºC</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TemperatureViewer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TemperatureViewer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupMethod"/>
 </buttongroups>
</ui>
